spec/models/schedule_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Schedule, type: :model do
+  describe 'validation' do
+    context '#from' do
+      it 'is not nil' do
+        subject.from = nil
+        subject.valid?
```

Comment:
It's exactly the same behaviour, but maybe more semantically correct to trigger this with `subject.validate` instead of the predicate.